src/soot/jimple/spark/solver/OnFlyCallGraph.java

```diff
@@ -72,15 +77,15 @@ public void build() {
     private void processReachables() {
         reachableMethods.update();
         while(reachablesReader.hasNext()) {
-            MethodOrMethodContext m = (MethodOrMethodContext) reachablesReader.next();
```

Comment:
My eclipse auto-remove this pointless cast. Although strictly not part of the functionality of this commit I see no reason to remove this?